protocol/contracts/libraries/Well/LibWell.sol

```diff
@@ -219,14 +219,22 @@ library LibWell {
         s.twaReserves[well].reserve1 = 1;
     }
 
-    function getWellPriceFromTwaReserves(address well) internal view returns (uint256 price) {
+    /**
+     * @notice returns the price in terms of TKN/BEAN. 
+     * (if eth is 1000 beans, this function will return 1000e6);
+     */
+    function getBeanTknPriceFromTwaReserves(address well) internal view returns (uint256 price) {
```

Comment:
Can you spell out “Token” instead of “Tkn”

Reply:
Fixed: d2a52383fcdf8985fb1ce59aeee0b04cbb5dadc1